install-script/install.sh

```diff
@@ -456,7 +456,20 @@ function get_latest_version() {
 
     # get latest version directly from website if there is no versions from api
     if [[ -z "${versions}" ]]; then
-        curl --retry 5 --connect-timeout 5 --max-time 30 --retry-delay 0 --retry-max-time 150 -s https://github.com/SumoLogic/sumologic-otel-collector/releases | grep -oE '/SumoLogic/sumologic-otel-collector/releases/tag/(.*)"' | head -n 1 | sed 's%/SumoLogic/sumologic-otel-collector/releases/tag/v\([^"]*\)".*%\1%g'
```

Comment:
We could change `head -n 1` to something else or grep out rc versions, or make `/SumoLogic/sumologic-otel-collector/releases/tag/(.*)` more specific

Reply:
made the change, please review